pkg/pr/prcreate.go

```diff
@@ -247,17 +248,19 @@ func createBody(exe utils.Executor, options *CreateOptions, commits string) (str
 	// CheckList
 	body = addDocSection(body, "## 📋 Checklist\n")
 
-	if options.NoLint {
-		body = addDocSection(body, "* ⛔ **This PR has not been linted!** ⚠️")
-		// TODO: Auto-Testing. If not auto-tested, ask why?
-	} else {
+	if pr.isLinted {
 		body = addDocSection(body, "* ✅ Lint checks passed on local machine.")
-	}
-	if options.NoUnit {
-		body = addDocSection(body, "* ⛔ **This PR has not been unit tested!** ⚠️")
-		// TODO: Auto-Linting. If not auto-linted, ask why?
+	} else if options.NoLint {
+		body = addDocSection(body, "* ⛔ **This PR has not been linted! The --nolint option was used.** ⚠️")
 	} else {
+		body = addDocSection(body, "* ⛔ **This PR has not been linted! Unspecified lint error!** ⚠️")
+	}
+	if pr.isTested {
 		body = addDocSection(body, "* ✅ Unit tests passed on local machine.")
+	} else if options.NoUnit {
+		body = addDocSection(body, "* ⛔ **This PR has not been unit tested! The --notest option was used.** ⚠️")
+	} else {
+		body = addDocSection(body, "* ⛔ **No tests could be run for this PR.**")
```

Comment:
Suggestion: Use the warning triangle instead of the red stop sign on the "no tests could be run". This matches the severity scale imo
Good: Tests pass
Medium: There are no tests
Bad: Tests were ignored